Signal/src/ViewControllers/MediaMessageView.swift

```diff
@@ -355,16 +353,16 @@ class MediaMessageView: UIView, OWSAudioAttachmentPlayerDelegate {
     }
 
     private func setAudioIconToPlay() {
-        let image = UIImage(named:"audio_play_black_large")?.withRenderingMode(.alwaysTemplate)
+        let image = UIImage(named: "audio_play_black_large")?.withRenderingMode(.alwaysTemplate)
         assert(image != nil)
-        audioPlayButton?.setImage(image, for:.normal)
+        audioPlayButton?.setImage(image, for: .normal)
         audioPlayButton?.imageView?.tintColor = UIColor.ows_materialBlue()
     }
 
     private func setAudioIconToPause() {
-        let image = UIImage(named:"audio_pause_black_large")?.withRenderingMode(.alwaysTemplate)
+        let image = UIImage(named: "audio_pause_black_large")?.withRenderingMode(.alwaysTemplate)
         assert(image != nil)
-        audioPlayButton?.setImage(image, for:.normal)
+        audioPlayButton?.setImage(image, for: .normal)
```

Comment:
I'm sorry but `clang-format` behaves differently on my laptop and desktop.

Reply:
This is the correct one =)